### ISSUE TITLE
can you report a single model MUTAN + Att performance?

### ISSUE THREAD

**Issue by gaopeng-eugene** [Cadene/vqa.pytorch#16]
In your paper, you report ensemble model. Because you are the state-of-the-art VQA method right now, can you report a single model MUTAN+Att performance? It is easier to compare with.

**Comment by Cadene** [comment]
You should be able to download a pretrained model (the open ended accuracy is included in the zip).

```
mkdir -p logs/vqa
cd logs/vqa
wget http://webia.lip6.fr/~cadene/Downloads/vqa.pytorch/logs/vqa/mutan_noatt_train.zip 
wget http://webia.lip6.fr/~cadene/Downloads/vqa.pytorch/logs/vqa/mlb_att_trainval.zip 
wget http://webia.lip6.fr/~cadene/Downloads/vqa.pytorch/logs/vqa/mutan_att_trainval.zip 
```

(The last one)

Our fusion scheme should be state of the art (as far as we know), but other fusion schemes are competitive (MCB, MLB, MFB). However multiple tricks and different features and architectures can be used outside the fusion scheme to improve the accuracy further more (please, look at the CVPR workshop about VQA).

**Comment by aronsar** [comment]
Hi, it seems like the link is dead now (for mutan_att_trainval), would you mind please posting another one?
